packages/botonic-plugin-hubtype-babel/src/index.ts

```diff
@@ -45,15 +47,22 @@ export default class BotonicPluginHubtypeBabel implements Plugin {
         }
 
         const text = request.input.data
-        const response = await this.apiService.inference(text, authToken)
+        const response = await this.apiService.inference(
+          text,
+          authToken,
+          this.hasSenseEnabled
+        )
 
         request.input.intent = response.data['intents'][0]['label']
         request.input.confidence = response.data['intents'][0]['confidence']
         request.input.intents = response.data['intents'].map(x => ({
           intent: x['label'],
           confidence: x['confidence'],
         }))
-        request.input.hasSense = response.data['has_sense']
+
+        if (this.hasSenseEnabled) {
+          request.input.hasSense = response.data['has_sense']
```

Comment:
so now the `input.hasSense` might be there or not, maybe this is undesirable and we prefer to put a `true` if no has_sense is provided by endpoint? or type properly this `input` to indicate this field might not exists